package.json

```diff
@@ -7,6 +7,7 @@
     "format": "sugarcube-2"
   },
   "scripts": {
+    "dotgraph": "gulp build && tweego -f dotgraph -o dist/index.html --head=src/head-content.html project",
```

Comment:
I think you want to output a different file from your actual game: 
`"dotgraph": "gulp build && tweego -f dotgraph -o dist/dotgraph.html --head=src/head-content.html project",`